src/full/Agda/Syntax/Internal.hs

```diff
@@ -884,8 +881,15 @@ dummyDom file line = defaultDom $ dummyType file line
 __DUMMY_DOM__ :: HasCallStack => Dom Type
 __DUMMY_DOM__ = withFileAndLine' (freezeCallStack callStack) dummyDom
 
+-- | Constant level @n@
+closedLevel :: Integer -> Level
+closedLevel n = Max n []
+
```

Comment:
May be worth defining as a pattern synonym (and maybe even one for `lzero` too)
given how often it is used on the LHS?